src/main/java/htsjdk/samtools/SAMSequenceDictionary.java

```diff
@@ -220,6 +223,9 @@ public boolean equals(Object o) {
      * <code>1,chr1,chr01,01,CM000663,NC_000001.10</code> e.g:
```

Comment:
I think the comment on the equals method above needs updating to say that alias's are not considered but Alternative Sequences are.